reflex/vars.py

```diff
@@ -120,7 +125,7 @@ class VarData(Base):
     state: str = ""
 
     # Imports needed to render this var
-    imports: ImportDict = {}
+    imports: Union[ImportDict, ParsedImportDict] = {}
```

Comment:
Should we just make this `imports: ParsedImportDict` and have the `imports` arg for `__init__` take the `Union[ImportDict, ParsedImportDict]`